lib/BackbeatConsumer.js

```diff
@@ -100,6 +106,23 @@ class BackbeatConsumer extends EventEmitter {
                         offset,
                     });
                     this.emit('error', err, entry);
+                } else if (entry.topic === 'backbeat-replication') {
```

Comment:
can you use the value from the config (`config.extensions.replication.topic`)?

Reply:
just a heads up for this, for now, I'll need to require the `Config` directly in this file. If it makes sense to, later on I can refactor by passing the crr topic directly to the consumer as a separate param

Reply:
OK for now, go for it.